media-collections/create-rename-delete-search-and-download-as-zip.md

```diff
@@ -0,0 +1,39 @@
+# Create, rename, delete, search & download media collections as zip
+
+You can create, rename, delete, search, and download media collections as zip files using ImageKit's dashboard. Learn more about permissions for these operations for restricted users [here](../collaboration-and-sharing/README.md#media-collection-permission-levels).
+
+## Create new media collection
+
+Navigate into the media collections page by clicking on "Media collections" in the menu bar. Once you are here, click on the "New" button in the top right corner. This would open a popup in which you can enter the name of the media collection and click on "Submit." A new media collection will be created. A media collection's name should be unique across the organization account and can consist of alphanumeric characters, underscores, and hyphens.
+
+Any restricted media library user can create a new media collection.
+
+## Rename media collection
+
+After you have created a media collection, you can rename it. Select the media collection that you want to rename and right-click on it to open the drop-down menu. Click on the "Rename" option; it will open a popup where you can enter the new name for the selected media collection. On clicking "Submit", the media collection will be renamed. The new name should be unique and can consist of alphanumeric characters, underscores, and hyphens.
+
+The assets added to a media collection or the users or user groups with whom it is shared are unaffected by renaming it. 
+
+## Delete media collections
+
+Right-click after selecting the media collections you want to delete. Click on the "Delete" option in the drop-down menu. This would open a confirmation popup. Click on "Submit" to permanently delete the media collections. Deleting a media collection does not delete the files and folders that have been added to it. Once deleted, all associated files and folders and their access to users and user groups through the media collection are removed.
+
+## Media collections search
+
+ImageKit provides you with a simple search functionality where you can search for a media collection by its name. On the media collections page, click on the search bar, type the full name or the characters at the beginning of the name, and hit enter.
+
+All media collections whose names either match exactly or begin with the provided input will be returned. The search for the media collection's name is case-insensitive.
+
+## Download media collections as zip
+
+ImageKit allows you to download multiple media collections and all of their contents, including files, folders, and sub-folders, as a single zip file. The downloaded zip maintains the original folder structure.
```

Comment:
I think `original folder structure` in above sentence is ambiguous. Is it actual folder location of those assets or the hierarchy a user sees when navigating that collection. I think later is true.